LetSwift/Helpers/Extensions/DictionaryExtension.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DictionaryExtension.swift
+//  LetSwift
+//
+//  Created by Sebastian Osiński on 10.11.2017.
+//  Copyright © 2017 Droids On Roids. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+extension Dictionary {
+
```

Comment:
Our code style guide recommends no additional line between extension and actual code, so the line 22 has to be removed.

BTW. Fantastic job, we are really happy you decided to become a part of our community, and we didn't expect any contribution that fast :) Our CI is still configured to work with Xcode 8, so it will always fail, but when we'll be ready to merge it, we will do this manually and then switch the stack to Xcode 9